example/src/test/kotlin/io/confluent/developer/e2e/SocketTestE2e.kt

```diff
@@ -0,0 +1,19 @@
+package io.confluent.developer.e2e
+
+import org.junit.jupiter.api.Test
+import org.testingisdocumenting.webtau.WebTauDsl.*
+import org.testingisdocumenting.webtau.junit5.WebTau
+
+@WebTau // optional for HTML report generation (can be set as a global extension)
+class SocketTestE2e {
```

Comment:
right now this test expect a server is running. Need help with gradle full pipeline. I usually use springboot runner from maven